csrankings-n.csv

```diff
@@ -326,8 +326,8 @@ Nicholas Kenelm Taylor,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs
 Nicholas Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos,Dvp7fNoAAAAJ
 Nicholas Mattei,Tulane University,http://www2.tulane.edu/sse/cs/faculty/nicholas-mattei.cfm,bUAYDkIAAAAJ
 Nicholas Miles Gibbins,University of Southampton,https://www.ecs.soton.ac.uk/people/nmg,umB6pHIAAAAJ
-Nicholas R. Jennings,Imperial College London,http://www.imperial.ac.uk/people/n.jennings,NOSCHOLARPAGE
-Nicholas Robert Jennings,Imperial College London,http://www.imperial.ac.uk/people/n.jennings,NOSCHOLARPAGE
+Nicholas R. Jennings,Loughborough University,https://www.lboro.ac.uk/services/vco/smt/vc-prof-jennings/,FX5CsuYAAAAJ
+Nicholas Robert Jennings,Loughborough University,https://www.lboro.ac.uk/services/vco/smt/vc-prof-jennings/,FX5CsuYAAAAJ

```

Comment:
This institution is not in the database; if you want to enter the CS department faculty for that institution, that'd be great, but you can also just delete this entry (and its aliases).

Reply:
country-info.csv -> line 136 Loughborough University 	europe 	uk